docs/content/deployment/run-coordinator.mdx

```diff
@@ -1,164 +1,140 @@
-
-##### Unblocking the queue
-
-If there are runs in progress that are blocking the queue, you have the option of terminating them via Dagit so that other runs can proceed.
-
-### Still stuck?
+The following run coordinators can be configured on your [Dagster instance](/deployment/dagster-instance):
 
-If these steps didn't help, reach out in Slack or file an issue and we'll be happy to help investigate.
+<table
+  className="table"
+  style={{
+    width: "100%",
+  }}
+>
+  <thead>
+    <tr>
+      <th
+        style={{
+          width: "20%",
+        }}
+      >
+        Term
+      </th>
+      <th>Definition</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <strong>
+          <PyObject
+            module="dagster._core.run_coordinator"
+            object="DefaultRunCoordinator"
+          />
+        </strong>
+      </td>
+      <td>
+        The{" "}
+        <PyObject
+          module="dagster._core.run_coordinator"
+          object="DefaultRunCoordinator"
+        />{" "}
+        calls <code>launch_run</code> on the instance’s run launcher immediately
+        in the same process, without applying any limits or prioritization
+        rules.
+        <br />
+        <br />
+        When this coordinator is set, clicking <strong>Launch Run</strong> in the
+        Dagster UI will immediately launch the run from the Dagit process. Similarly,
+        scheduled runs will immediately launch from the scheduler process.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>
+          <PyObject
+            module="dagster._core.run_coordinator"
+            object="QueuedRunCoordinator"
+          />
+        </strong>
+      </td>
+      <td>
+        The{" "}
+        <PyObject
+          module="dagster._core.run_coordinator"
+          object="QueuedRunCoordinator"
+        />{" "}
+        sends runs to the Dagster daemon via a run queue. The daemon pulls runs
+        from the queue and calls <code>launch_run</code> on submitted runs.
+        <br />
+        <br />
+        Using this run coordinator enables <a href="/guides/limiting-concurrency-in-data-pipelines">
+          instance-level limits on run concurrency
+        </a>, as well as <a href="/guides/customizing-run-queue-priority">
+          custom run prioritization rules
+        </a>.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Custom run coordinator</strong>
+      </td>
+      <td>
+        Using a subclass of one of the <code>RunCoordinator</code>s, you can
+        define a custom queueing strategy when submitting runs to the{" "}
+        <a href="/deployment/dagster-daemon">Dagster daemon</a> or executing
+        custom hooks before runs are submitted. For example, you can read HTTP
+        headers to inject run tags for{" "}
+        <a href="/guides/dagster/run-attribution">run attribution</a>.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+---
+
+## Configuring run coordinators
```

Comment:
This section is the only content that has really changed in this guide. Everything else is cosmetic.

I'm not sure it's 100% accurate, so eyeballs here would be great.